src/main/kotlin/no/nav/familie/klage/brevmottaker/domain/NyBrevmottaker.kt

```diff
@@ -49,9 +54,18 @@ data class NyBrevmottakerPersonUtenIdent(
 
 data class NyBrevmottakerPersonMedIdent(
     val personIdent: String,
-    val mottakerRolle: MottakerRolle,
-    val navn: String,
-) : NyBrevmottaker
+    override val mottakerRolle: MottakerRolle,
+    override val navn: String,
+) : NyBrevmottakerPerson {
+    init {
+        if (personIdent.isBlank()) {
+            throw IllegalStateException("Personident kan ikke være blank.")
+        }
+        if (navn.isBlank()) {
+            throw IllegalStateException("Navn kan ikke være blank.")
+        }
+    }
+}
```

Comment:
```suggestion
    init {
        require(personIdent.isNotBlank()) { "Personident kan ikke være blank." }
        require(navn.isNotBlank()) { "Navn kan ikke være blank." }
    }
}
```

Lurer på om det er mer clean med require her?

Reply:
Lurer samtidig på om man bør gjøre det samme for NyBrevmottakerPersonUtenIdent øverst, det er jo en gigantisk bølge av ifer og thrower 😛 
<img width="1240" alt="require" src="https://github.com/user-attachments/assets/a1d273eb-1ad0-4036-8f4b-da2b0bee3722" />